test/passenger_test.rb

```diff
@@ -69,6 +74,78 @@
   end
 
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+    before do
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+        )
+
+    end
+    it "returns Integer" do
+      # Arrange
+      td = RideShare::TripDispatcher.new
+      # Act & Assert
+      td.passengers.each do |passenger|
+        expect(passenger.net_expenditures).must_be_kind_of Integer
+      end
+    end
+    it "calculates the total amount of money a passenger has spent on their trips" do
+      td = RideShare::TripDispatcher.new
+      passenger = td.passengers[0]
+      expect(passenger.net_expenditures).must_equal 15
+    end
+    it "returns 0 when trips is empty array" do
+      expect(@passenger.net_expenditures).must_equal 0
+    end
+    
+  end
+
+  describe "total_time_spent" do 
+
+    before do
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+        )
+      trip = RideShare::Trip.new(
+        id: 8,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 9),
+        rating: 5,
+        driver: RideShare::Driver.new(
+          id: 2,
+          name: "Dan",
+          vin: "1B6CF40K1J3Y74UY2",
+          status: :AVAILABLE
+        )
+        )
+
+      @passenger.add_trip(trip)
+    end
+
+    it "returns int" do
+      # Arrange
+      td = RideShare::TripDispatcher.new
+      # Act & Assert
+      td.passengers.each do |passenger|
+        expect(passenger.total_time_spent).must_be_kind_of Integer
+      end
+    end
+
+    it "calculates the total amount of money a passenger has spent on their trips" do
+      td = RideShare::TripDispatcher.new
+      passenger = td.passengers[0]
+      expect(passenger.total_time_spent).must_equal 5410
+    end
+
+    it "returns 0 when trips is empty array" do
+      expect(@passenger.total_time_spent).must_equal 0
+    end
+    
```

Comment:
These tests look good. You should test for one more edge case when the passenger has in progress trip.